README.md

```diff
@@ -136,8 +135,14 @@ Non-Technical Requirements:
 
 ### Workspace Setup
 
-To be able to run **Terasology** from source, you'll need to setup your workspace.
-Follow the [Contributor Quick Start Guide](https://terasology.org/Terasology/#/Contributor-Quick-Start).
+To be able to run **Terasology** from source, you'll need to setup your workspace. For a quick setup execute the following:
+```
+./groovyw module init iota
+./gradlew game
+
+```
+
+For more info, follow the [Contributor Quick Start Guide](https://terasology.org/Terasology/#/Contributor-Quick-Start).
```

Comment:
Please revert this as it duplicates instructions from the contributor quick start which makes things harder to maintain without adding much benefit.